packages/mdc-side-sheet/common.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "@material/animation/functions";
+@import "@material/animation/variables";
+@import "@material/rtl/mixins";
+@import "@material/theme/mixins";
+@import "@material/theme/variables";
+@import "@material/typography/mixins";
+@import "@material/typography/variables";
+@import "./mixins";
+@import "./variables";
+
+// postcss-bem-linter: define side-sheet
+
+.mdc-side-sheet {
+  @include mdc-side-sheet-base_;
+  @include mdc-side-sheet-stroke-width_(1px);
+  @include mdc-side-sheet-stroke-color($mdc-side-sheet-stroke-color);
+  @include mdc-side-sheet-surface-fill-color($mdc-side-sheet-surface-fill-color);
+  @include mdc-side-sheet-shape-radius(large);
+  @include mdc-side-sheet-z-index($mdc-side-sheet-z-index);
+  @include mdc-side-sheet-width($mdc-side-sheet-width);
+  @include mdc-typography-base;
+
+  display: flex;
```

Comment:
Related to previous comment, as much of this as possible should be within the `base_` mixin (including the open/opening/closing/content stuff) if it will be the same for Drawer, so that Drawer can just include these mixins and have a minimum of additional styles only pertinent to Drawer.

Same goes for the dismissible and modal files/mixins